app/editor/src/features/navbar/NavBar.tsx

```diff
@@ -45,6 +45,11 @@ export const NavBar: React.FC = () => {
               label="CBRA Report"
               claim={Claim.administrator}
             />
+            <NavBarItem
+              navigateTo="/admin/media/types"
```

Comment:
Just an example of where you've used constant sometimes and not others.  I think we probably shouldn't use constant values for paths.  Instead just have props with default values that we can override if required.